CMakeLists.txt

```diff
@@ -482,10 +482,13 @@ if(WITH_OPENTRACING)
     set(OPENTRACING_DIR "third_party/opentracing-cpp")
     message("Trying to use local ${OPENTRACING_DIR} from submodule")
     if(EXISTS "${PROJECT_SOURCE_DIR}/${OPENTRACING_DIR}/.git")
+      set(SAVED_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+      set(CMAKE_CXX_FLAGS OFF)
```

Comment:
Why do we need set `CMAKE_CXX_FLAGS` to `OFF` here?
If we use `-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0` , I think it's also need be passed into all targets.